docs/source/advanced/backends.rst

```diff
@@ -1,14 +1,108 @@
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | atomicOp                                                      | not necessary                                 | std::lockguard< std::mutex >                                                  | n/a                                                                          | #pragma omp critical                                                            | #pragma omp critical                                                                                                               | atomicXXX                                        |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+    | ``ALPAKA_FN_HOST_ACC``, ``ALPAKA_FN_ACC``, ``ALPAKA_FN_HOST`` | inline                                        | inline                                                                        | inline                                                                       | inline                                                                          | inline                                                                                                                             | __device__, host, forceinline                    |
+    +---------------------------------------------------------------+-----------------------------------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------+---------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------+
+
+Serial
+``````
+
+The serial accelerator only allows blocks with exactly one thread.
```

Comment:
Is that correct? I thought it allows any workdiv, just executes sequentially

Reply:
Good question. I never tried it. I simply copy and past the text. @psychocoderHPC Do you know, if it is (still) correct?

Reply:
The comment is true. You can not have multiple threads in a block because you can not synchronize those threads because they can not be executed asynchronously.

Reply:
Ah, true. Thanks for explaining.